dspace/config/registries/dublin-core-types.xml

```diff
@@ -268,6 +268,196 @@
     <qualifier>uri</qualifier>
     <scope_note>Uniform Resource Identifier</scope_note>
   </dc-type>
+  
+  <!-- OpenAIRE4 Guidelines -->
+  <!-- 2.2.2.1. Attribute nameType (R) -->
+  <!-- and 3.2.6. Subproperty nameIdentifier (R) -->
+  <!-- specific type required by OpenAIRE4 Guidelines -->  
+  
+  <dc-type>
+    <schema>dc</schema>
+    <element>identifier</element>
+    <qualifier>authorid</qualifier>
+    <scope_note>Scopus Author ID</scope_note>
```

Comment:
A general comment on *all* these new `dc.identifier` fields.  I wonder if we should move these each to the schema that most aligns with the identifier in question?  For example, this `dc.identifier.authorid` only applies to `Person` Entities.  So, we might think to move this to the `person` schema, as Schema.org has its own Identifier field (which applies to all Types): https://schema.org/identifier

Additionally, if you read the ["background" info on `identifier` from Schema.org](https://schema.org/docs/datamodel.html#identifierBg), they say:
> In the most complex case, there is sometimes a need to represent the type of an identifier. In this case, a PropertyValue pair ('name', 'identifier') pair can be used when a standard URI form of the identifier is unavailable.

The above statement _implies_ to me that we can essentially create qualifiers on their `identifier` which provide the type of the identifier.  This seems to mean that `person.identifier.scopusid` (I'd prefer we rename this `scopusid`) would be a perfectly reasonable representation of this Person Identifier in Schema.org.

If we all agree to this approach, I suspect that *most* of the below `dc.identifier.*` fields could be moved to the appropriate Schema.org type instead of being in our "dc" schema.

Reply:
I talked with José and we agree that your suggestion doesn't move away from the initial purposes  of this PR. We reverted the changes and move the identifiers into specific schemas.
My only concern is the usage of the:

> PropertyValue pair ('name', 'identifier')

as a qualifier. Schema.org isn't clear about the usage when I first read it, I though the representation of that should be in the value of the *.identifier. Something like: person.identifier=doi:10.123123/12345. But of course, we can give this our interpretation and use the qualifier.